content/events/seminar-series/_index.md

```diff
@@ -0,0 +1,37 @@
++++
+title = "Nordic-RSE seminar series"
+description = "Nordic RSE – Research Software Engineers in the nordics – seminar series"
+template = "section-with-toc.html"
+[extra]
+subtitle = ""
++++
+
+Are you developing software or tools that are driven by research/engineering in either academia or industry? 
+
+Need to network, share knowledge and experiences with your peers? 
+
+Then you came to the right page: Nordic-RSE invites everyone interested to participate in Nordis-RSE seminars:
+
+The series will start out as a **monthly event in August 2021**. The time for each seminar will be set by the speaker. We are inviting everyone to suggest topics and/or speakers for this series (link coming soon).
+
+To kick off the event some of our members will introduce you to their work at Nordic universities.
+
```

Comment:
```suggestion

## Format

As an example, events can be 60 (40+20 discussion) or 30 (15+15 discussion) minutes.  Events could be talks, demos, discussion, debates, and so on.  Events are hosted with HackMD for public anynchronous discussion.  We will provide mentoring for speakers.

```

Reply:
Good idea to add sub-headers by the way. We could maybe add some more, and use other ways to delimit/highlight the text.

Reply:
Mini symposium could be great as well. Maybe we can leave the number of talks per seminar open and see how it goes and adjust accordingly?
Yes more subheaders and other highlighters could be added, will take a look at it